packages/unmock-core/src/service/spy/generated/index.ts

```diff
@@ -3,24 +3,65 @@
  */
 import { match, SinonMatcher } from "sinon";
```

Comment:
Remind me why we're checking this into VC? I'm sure there's a good reason, but I forget what the decision was.

Reply:
I think it's just nicer to manage, changing it is a conscious operation instead of requiring repo pullers to generate that themselves as part of the build.